tests/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "type": "module",
+  "private": "true",
```

Comment:
Now the private attribute is duplicated 😂.
This changed in the main branch recently.

Reply:
:smiling_face_with_tear: